### PR TITLE
added admin service page

### LINKED ISSUE #133: Add Services tab in admin sidebar for all manipulations related to services
**Is your feature request related to a problem? Please describe.**

**Additional context**
Are you working on this?
Yes

### PR DESCRIPTION
**Pull request by ashwaniYDV**
Fixes #133 
Changes: [Add here what changes you made in this Pull Request and if possible provide links depicting the changes.]

Surge Deployment Link: https://pr-154-sahaayak.surge.sh

Screenshots for the change: [Add here screenshots showing the changes.]

### CODE REVIEW

**Comment by ashwaniYDV** [comment]
@harsh-jindal Bhaiya, I don't know why buid error is coming. It's not showing on the console. It's working fine

File: src/components/Admin/Users.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Button, Table, Tag, notification, Input, Icon, Alert } from 'antd';
 import Highlighter from 'react-highlight-words';
 
 import { getAllUsers, deleteUser, deleteAllServices } from "../../redux/actions/adminActions";
-import './users.css';
+import './Users.css';
```
Comment (harsh-jindal) [inline src/components/Admin/Users.js:8]: @ashwaniYDV  File Name is `users.css` not `Users.css`
This is why travis is failing

Comment (ashwaniYDV) [reply src/components/Admin/Users.js:8]: @harsh-jindal but I changed the file name to Users.css to keep it consistent with other files names.

Comment (harsh-jindal) [reply src/components/Admin/Users.js:8]: @ashwaniYDV PLease check in files changed what files have been changed. You have not changed `users.css` to `Users.css`

Comment (ashwaniYDV) [reply src/components/Admin/Users.js:8]: @harsh-jindal I have changed Users.css to users.css still Travis is failing. Please help. There is no such error in my console

**Pull request closed without merging**